bigtable-dataflow-parent/bigtable-beam-import/src/main/resources/log4j.properties

```diff
@@ -22,3 +22,5 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 
 log4j.category.org.apache.hadoop.io.compress.CodecPool=WARN
+log4j.logger.org.apache.hadoop.io.compress.Compression=TRACE
+log4j.logger.org.apache.hadoop.util.NativeCodeLoader=DEBUG
```

Comment:
are these levels intentional?

Reply:
Added by import PR. Not part of the synctable PR.